src/crimson/osd/shard_services.h

```diff
@@ -0,0 +1,129 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#pragma once
+
+#include <seastar/core/future.hh>
+
+#include "crimson/os/cyan_store.h"
+#include "crimson/osd/osdmap_service.h"
+#include "crimson/mgr/client.h"
+#include "crimson/mon/MonClient.h"
+#include "crimson/net/Messenger.h"
```

Comment:
ditto.

Reply:
I can actually get rid of all of these by moving PeeringCtx and Collection out of their respective classes, will do.